internal/slicer/slicer_test.go

```diff
@@ -524,14 +522,6 @@ var slicerTests = []slicerTest{{
 		"/dir/":       "dir 0755",
 		"/other-dir/": "dir 0755",
 	},
-	report: map[string]string{
-		// TODO These first three entries should be removed from the report.
-		"/dir/nested/":           "dir 0755 {test-package_myslice}",
-		"/dir/nested/file":       "file 0644 84237a05 {test-package_myslice}",
-		"/dir/nested/other-file": "file 0644 6b86b273 {test-package_myslice}",
-
-		"/other-dir/text-file": "file 0644 5b41362b {test-package_myslice}",
-	},
```

Comment:
Oh, we are only removing the `report` because the `report` would be empty. The reason being all the paths have `until: mutate`  specified the filesystem only contains the implicit parent dirs which we are not adding to our report either.